src/plugins/common/table/lib/table-plugin.js

```diff
@@ -853,6 +855,83 @@ define([
 				}
 			}
 		});
+
+		const haveCellConfig = Array.isArray(that.settings.cellConfig) && that.settings.cellConfig.length > 0;
+		const haveColumnConfig = Array.isArray(that.settings.columnConfig) && that.settings.columnConfig.length > 0;
+		const haveRowConfig = Array.isArray(that.settings.rowConfig) && that.settings.rowConfig.length > 0;
+
+		if (haveCellConfig || haveColumnConfig || haveRowConfig) {
+			this._tableCellStyleButton = Ui.adopt('tableCellStyle', ContextToggleButton, {
+				contextType: 'dropdown',
+				tooltip: i18n.t("button.cellstyle.tooltip"),
+				icon: Icons.TABLE_STYLE_CELLS,
+				context: function () {
+					const options = [];
+					const selType = TablePlugin.activeTable.selection.selectionType;
+					const usedClasses = new Set();
+
+					if (haveCellConfig) {
+						for (let i = 0; i < that.settings.cellConfig.length; i++) {
```

Comment:
In function auslagern: `function xyx(config, currentOptions)`
```js
const currentOptions = {};

function xyz(config, currentOptions) {
  if (currentOptions[config[0].id]) {
    return;
  }
}

Object.values(xyz([], currentOptions);
```

Reply:
Done